ubuntu

```diff
@@ -7,7 +7,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-       make git g++ gfortran ccache vim clang llvm wget sudo curl ninja-build clang-format zstd libhwloc-dev libomp-dev gnupg2 cmake && \
+       make git g++ gfortran ccache vim clang llvm wget sudo curl ninja-build clang-format zstd libhwloc-dev libomp-dev gnupg2 cmake libgtest-dev clang-tidy gcc-multilib g++-multilib gfortran-multilib && \
```

Comment:
What do we need `multilib` for?  Is that about cross-compiling?

Reply:
For the 32-bit workflow.

Reply:
I just did a `git grep apt.*install` .github/` in the kokkos repo.